packages/expo-secure-store/android/src/main/java/expo/modules/securestore/EncryptionCallback.kt

```diff
@@ -0,0 +1,18 @@
+package expo.modules.securestore
+
+import expo.modules.core.Promise
+import org.json.JSONException
+import java.security.GeneralSecurityException
+import javax.crypto.Cipher
+import javax.crypto.spec.GCMParameterSpec
+
+// Interface used to pass encryption/decryption logic
+interface EncryptionCallback {
```

Comment:
```suggestion
fun interface EncryptionCallback {
```

Reply:
Done in https://github.com/expo/expo/pull/14512/commits/531d1a47ddd80d2b5758771176f205a0c590ca9a.